test/e_chronicler_web/controllers/journal_entry_controller_test.exs

```diff
@@ -20,14 +22,33 @@ defmodule EChroniclerWeb.JournalEntryControllerTest do
   test "Single Journal Entry Is Displayed At /:id", %{conn: conn} do
     {:ok, journal_entry} = EChronicler.Models.JournalEntry.create_journal_entry(%{author: "Bob", title: "Hello", entry: "Test."})
     {:ok, other_journal_entry} = EChronicler.Models.JournalEntry.create_journal_entry(%{author: "Betty", title: "Goodbye", entry: "Test."})
-    conn = get(conn, "/#{journal_entry.id}")
+    conn = get(conn, "/post/#{journal_entry.id}")
     assert html_response(conn, 200) =~ journal_entry.title
     refute html_response(conn, 200) =~ other_journal_entry.title
   end
 
   test "Return 404 if Entry Is Not Found at /:id", %{conn: conn} do
-    conn = get(conn, "/5")
+    conn = get(conn, "/post/5")
     assert html_response(conn, 404)
   end
 
+
+
+  test "renders form", %{conn: conn} do
+    conn = get(conn, "/new")
+    assert html_response(conn, 200) =~ "Write New Post"
+  end
+
+
+  test "Redirects to show journal entry when data is valid", %{conn: conn} do
+    conn = post(conn, Routes.journal_entry_path(conn, :create), journal_entry: @valid_journal_entry)
+
+    assert %{id: id} = redirected_params(conn)
+    assert redirected_to(conn) == Routes.journal_entry_path(conn, :show, id)
+
+    conn = get(conn, Routes.journal_entry_path(conn, :show, id))
+    assert html_response(conn, 200) =~ "Awesome Blog Post"
```

Comment:
You have "flash" behavior in your controller for submission but you don't test it.